### DISCUSSION TITLE
Empty nav__title

### DISCUSSION THREAD

**Discussion by blogi-kriptovalyut** [mmistakes/minimal-mistakes#2668] [Q&A] [answered]
Hey. I am using `minimal-mistakes "4.16.4"`.
Please tell me if the behavior of `jekyll-toc` has been fixed in new versions?
I mean, if the table of contents is empty, then you don't need to output an empty `nav__title`.

![img-2020-08-24-11-53-22](https://user-images.githubusercontent.com/50487083/91024628-e5528380-e600-11ea-85d3-5feb7497a6fb.png)

![img-2020-08-24-11-54-54](https://user-images.githubusercontent.com/50487083/91024718-061ad900-e601-11ea-8b1a-3be20791794e.png)

I know that I can just turn off the `toc: true` option, but it would be very convenient to specify in `_config.yml` so that `toc: true` is always present, but if it is empty, then do not show it.
Is it possible?

I like Minimal Mistakes more and more ;)

**Comment by iBug** [comment]
Just a quick note: You can always specify `toc: false` in the [front matter](https://jekyllrb.com/docs/front-matter/) of individual pages, which will override the "global" setting in `_config.yml`.

**Comment by mmistakes** [comment]
>  it would be very convenient to specify in `_config.yml` so that `toc: true` is always present

You can use [Jekyll's front matter defaults setting](https://jekyllrb.com/docs/configuration/front-matter-defaults/) to configure `toc: true` for all. Then as @iBug suggested on the pages you want it disabled add `toc: false`.

**Comment by blogi-kriptovalyut** [comment]
Ok, I know this, I thought that automatically don't show an empty `nav__title` if there are no subheadings. Without `toc: false`

**Comment by mmistakes** [comment, marked as answer]
Doesn't work that way. The nav title doesn't know if the page has subheadings or not that are used by jekyll-toc. Unfortunately you need to manually turn it on or off.
